spec/factories/restaurants.rb

```diff
@@ -0,0 +1,35 @@
+require 'factory_bot'
+
+FactoryBot.define do
```

Comment:
YES!
Great builder! Hope it saved you the gritty at the tests. Did you use all the traits (I couldn't see them at the test)? Where they helpful?